src/panels/lovelace/cards/hui-entity-filter-card.ts

```diff
@@ -26,8 +26,8 @@ class EntityFilterCard extends UpdatingElement implements LovelaceCard {
 
   private _oldEntities?: EntityFilterEntityConfig[];
 
-  public getCardSize(): number {
-    return this._element ? computeCardSize(this._element) : 1;
+  public async getCardSize(): Promise<number> {
+    return this._element ? await computeCardSize(this._element) : 1;
```

Comment:
We can do a micro optimization here by bypassing a promise if we don't have an element. Would that be beneficial? 

```suggestion
  public getCardSize(): number | Promise<number> {
    return this._element ? computeCardSize(this._element) : 1;
```